tests/tools.vitruv.applications.umljava.tests/META-INF/MANIFEST.MF

```diff
@@ -5,9 +5,16 @@ Bundle-SymbolicName: tools.vitruv.applications.umljava.tests
 Automatic-Module-Name: tools.vitruv.applications.umljava.tests
 Bundle-Version: 0.2.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-11
-Fragment-Host: tools.vitruv.applications.umljava
```

Comment:
`umljava.tests` is also no longer a fragment to allow proper dependencies on it.

Reply:
This is fine for now and should be reverted as soon as we finally cleanup all the uilities (see #3).